src/it/tita/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.mule.modules</groupId>
-    <artifactId>mule-json-module</artifactId>
+    <artifactId>mule-json-module-tita-test</artifactId>
```

Comment:
I don't think we need to rename this after all this the name of the mule app no ?

Reply:
Yes, I know it is not necessary but I thought it might be confusing to use the same artifact name for the connector and for the tita app.